content/en/logging/cert-transparency-log-info.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "Certificate transparency log information"
+category: "Transparency Log"
```

Comment:
This should be under Certificate Authority, not Transparency Log

Reply:
OK, will move